src/fu-util-common.c

```diff
@@ -1702,6 +1702,10 @@ fu_util_plugin_flag_to_string(FwupdPluginFlags plugin_flag)
 		/* TRANSLATORS: the plugin was created from a .so object, and was not built-in */
 		return _("Loaded from an external module");
 	}
+	if (plugin_flag == FWUPD_PLUGIN_FLAG_VERIFY_SYSTEM_INTEGRITY) {
+		/* TRANSLATORS: check various UEFI and ACPI tables are unchanged after the update */
+		return _("Will verify system integrity after update");
```

Comment:
Will capture elements of system integrity around an update.

Reply:
How about `FWUPD_PLUGIN_FLAG_MEASURE_SYSTEM_INTEGRITY` too?

Reply:
I did that :)